src/NServiceBus.Core/Encryption/EncryptedValue.cs

```diff
@@ -6,16 +6,28 @@ namespace NServiceBus
     /// Class used to represent an encrypted value with an initialization vector.
     /// </summary>
     [Serializable]
+    [ObsoleteEx(
+        Message = "Message property encryption is released as a dedicated 'NServiceBus.Encryption.MessageProperty' package.",
+        RemoveInVersion = "8",
+        TreatAsErrorFromVersion = "7")]
     public class EncryptedValue
     {
         /// <summary>
         /// The encrypted value represented as a Base64 string.
         /// </summary>
+        [ObsoleteEx(
+            Message = "Message property encryption is released as a dedicated 'NServiceBus.Encryption.MessageProperty' package.",
```

Comment:
Not sure it is strictly necessary. But since we are using everywhere the same text would it make sense to use a constant?